warehouse/query-core/src/test/java/datawave/query/planner/AnyFieldQueryPlannerNoFieldExpansionTest.java

```diff
@@ -0,0 +1,337 @@
+package datawave.query.planner;
+
+import datawave.query.Constants;
+import datawave.query.exceptions.DatawaveFatalQueryException;
+import datawave.query.exceptions.FullTableScansDisallowedException;
+import datawave.query.testframework.AbstractFunctionalQuery;
+import datawave.query.testframework.AccumuloSetupHelper;
+import datawave.query.testframework.CitiesDataType;
+import datawave.query.testframework.CitiesDataType.CityEntry;
+import datawave.query.testframework.CitiesDataType.CityField;
+import datawave.query.testframework.DataTypeHadoopConfig;
+import datawave.query.testframework.FieldConfig;
+import datawave.query.testframework.GenericCityFields;
+import org.apache.log4j.Logger;
```

Comment:
SLF4j?

Reply:
I thought we were using log4j everywhere.... did I miss something?

Reply:
In terms of our tests, everything appears to be using log4j instead of slf4j.  The only exception is QueryExceptionTest in the base-responses service.  leaving log4j for now